leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -345,7 +345,7 @@ private void proposeLeadership(LeadershipToken token) {
             proposer.propose(seq, null);
         } catch (PaxosRoundFailureException e) {
             // We have failed trying to become the leader.
-            leaderLog.warn("Leadership was not gained", e);
+            leaderLog.warn("Leadership was not gained");
```

Comment:
Sorry, was thinking out loud and got carried away. To summarize, (1) I'm aware that even if we include information about which round failed, the root cause is still difficult to understand so the logging in this PR is still not perfect, but (2) we are trying to address this as part of a separate refactor, and (3) in the meantime, I will still log which phase failed in the spirit of not throwing away data.